t/Currency.t

```diff
@@ -111,13 +111,19 @@ ddg_spice_test(
         caller => 'DDG::Spice::Currency',
         is_cached => 0
     ),
-    # using value keyword
+    # using value, price keyword
     'bitcoin value' => test_spice(
         '/js/spice/currency/1/xbt/xbt',
         call_type => 'include',
         caller => 'DDG::Spice::Currency',
         is_cached => 0
     ),
+    'bitcoin price' => test_spice(
+        '/js/spice/currency/1/xbt/xbt',
+        call_type => 'include',
+        caller => 'DDG::Spice::Currency',
+        is_cached => 0
+    ),
```

Comment:
I can't get either of these tests to run in the search bar for some reason. Can you?